tests/projects/AzureFunctionProject/Function1.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+
+namespace StrykerReplication
```

Comment:
Use global namespace;

Reply:
Allign the namespace with the project file, unless there is a reason not to so. (And that might then require an explanation).

Reply:
done, thanks for the feedback